### PR TITLE
S01 us01

### PR DESCRIPTION
**Pull request by codelande**

### CODE REVIEW

File: src/View/layout.html.twig [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
-</head>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link
+		rel="icon" href="/assets/images/favicon.png">
+		<!-- Bootstrap CSS -->
+		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+		<link
```
Comment (MATKYU) [inline src/View/layout.html.twig:11]: SPACE

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
+		rel="stylesheet" href="assets/css/style.css">
+		<!-- Bootstrap JS (with defer) -->
+		<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous" defer></script>
+		<title>
+			{% block title %}Rigolou
```
Comment (MATKYU) [inline src/View/layout.html.twig:16]: Placement des blocks

File: src/View/layout.html.twig [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous" defer></script>
-    <title>{% block title %}Rigolou{% endblock %}</title>
-    {% block stylesheet %}{% endblock %}
-</head>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link
+		rel="icon" href="/assets/images/favicon.png">
```
Comment (MATKYU) [inline src/View/layout.html.twig:8]: Space

File: src/View/layout.html.twig (hunk 2 above)
Comment (MATKYU) [inline src/View/layout.html.twig:16]: Block placement

File: src/View/layout.html.twig [hunk 4]
```diff
@@ -1,25 +1,29 @@
...
+		<!-- Bootstrap CSS -->
+		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+		<link
+		rel="stylesheet" href="assets/css/style.css">
```
Comment (MATKYU) [inline src/View/layout.html.twig:12]: Space

**Review by MATKYU — CHANGES_REQUESTED** [review]

**Review by MATKYU — APPROVED** [review]

**Review by sblondeau — DISMISSED** [review]

**Review by MGliem — APPROVED** [review]

**Pull request merged**